Home page prompts user to login to view data, public or controlled-access

mjkrause opened the pull request.

Commit:
Removed if condition, and left text for controlled-access data

mjkrause commented:
@coverbeck, you can see that change at https://mkrause6.ucsc-cgp-dev.org.

coverbeck commented on templates/index.html:
```diff
@@ -24,25 +24,16 @@ <h2 class="center">Explore and Export Genomics Research Data</h2>
-                    log in, although that will not the change the functionality.
-                </p>
-            {% endif %}
+            <h3 class="center">Restricted Access Data</h3>
```
This doesn't seem right... The dev deployment will not require you to be whitelisted, it will only require you to be logged in. And that instance does not have restricted access data.

Basically, you still need a version of lines 41-43, although the part where it says you can view it anonymously needs to be modified.

mjkrause replied:
You're right. I've changed the wording to reflect this. Note, the running instance mkrause6.ucsc-cgp-dev.org doesn't show it as I use to work on #91.

Commit:
Corrected text under "Open access data"

Commit:
Changed wording

coverbeck commented on templates/index.html:
```diff
@@ -23,26 +23,10 @@ <h2 class="center">Explore and Export Genomics Research Data</h2>
-                    log in, although that will not the change the functionality.
-                </p>
-            {% endif %}
+            <h3 class="center">Open Access Data</h3>
```
You need the `if {% if auth_required %}` that you deleted, and the block inside it should remain unchanged. Your changes should go inside the `{% else %}` block.

Regarding the actual text, I would not say `with Google`, as we may someday change the log in to use Auth0 or Fence. You could update the text every time we change the provider, but I think it's easier to say to just log in and not get into the details of what the login implementation is.

I would also say `in order to browse it`, and leave out the `or export it`.

My suggestion would be something like: `You are required to log in order to browse the open access data.`

coverbeck replied:
The variable name `auth_required` may be part of the confusion, because now we are always requiring auth. Perhaps it would be clearer if you renamed that variable to something like `whitelist_validation_required`, as that is what that block indicates.

mjkrause replied:
I've put the if-condition back in, and changed the variable name to the one you suggested. Added text accordingly. Haven't tested it it in a running instance as I'm  having problems deploying, which appears to be caused by installing Oracle-Java in the dcc-dasboard-service container.

Commit:
Put back if-condition and changed name of variable

Put back the if condition as it is needed, with
the text slightly changed to reflect the conditions. I
changed the name of the variable from `auth_required`
to `whiltelist_validation_required`.

coverbeck commented on templates/index.html:
```diff
@@ -23,24 +23,21 @@ <h2 class="center">Explore and Export Genomics Research Data</h2>
-                    log in, although that will not the change the functionality.
+                    Although the data are open access you need
+                    to log in in order to browse it. To log in, click the login
+                    link at the top of the page
```
Missing a period at the end of the sentence. :)

mjkrause replied:
DOne. I also changed the title of this PR.

coverbeck reviewed (changes requested):
One small change...

Commit:
Added full-stop to sentence.
